curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-offer-technical-support-and-guidance/6620d3003ddaf50aac1bc76d.md

```diff
@@ -0,0 +1,54 @@
+---
+id: 6620d3003ddaf50aac1bc76d
+title: Task 16
+challengeType: 19
+dashedName: task-16
+---
+
+# --description--
+
+A `statement` in programming is a line of code that performs a specific action. For example, `print("Hello World")` is a statement that displays text.
+
+A `condition` is a test in code that checks if something is true or false. For instance, `if x > 5` checks if "x" is greater than 5.
+
+A `conditional` is generally an adjective referring to something that depends on a `condition` or `conditions`. In programming, it is used to describe any expression or feature that involves logical `conditions`. It's not limited to explicit "if-then" structure but can also refer to any setup that alters behavior based on `conditions`, including loops or even ternary operators.
+
+A `conditional statement` is a statement that only runs if a certain `condition` is true. For example, `if x > 5: print("Large")` is a `conditional statement` because it prints "Large" only if "x" is greater than 5.
```

Comment:
```suggestion
A `conditional statement` is a statement that only runs if a certain `condition` is true. For example, `if x > 5: print("Large")` is a `conditional statement` because it prints `Large` only if `x` is greater than 5.
```

I think we have to add backticks to "Large" and "x"